db/change_listener.go

```diff
@@ -212,10 +221,49 @@ func (listener *changeListener) Notify(ctx context.Context, keys channels.Set) {
 	}
 	base.DebugfCtx(ctx, base.KeyChanges, "Notifying that %q changed (keys=%q) count=%d",
 		base.MD(listener.bucketName), base.UD(keys), listener.counter)
-	listener.tapNotifier.Broadcast()
 	listener.tapNotifier.L.Unlock()
 }
 
+func (listener *changeListener) BroadcastChanges(ctx context.Context) {
```

Comment:
Rename to make this more obvious that it is starting a goroutine/process

```suggestion
func (listener *changeListener) StartNotifierBroadcaster(ctx context.Context) {
```